src/estimators.jl

```diff
@@ -61,13 +60,13 @@ mutable struct LGBMRegression <: LGBMEstimator
 
     metric::Vector{String}
     metric_freq::Int
-    is_training_metric::Bool
+    is_provide_training_metric::Bool
     eval_at::Vector{Int}
 
     num_machines::Int
     local_listen_port::Int
     time_out::Int
-    machine_list_file::String
+    machine_list_filename::String
 
     num_class::Int
```

Comment:
`num_class` shouldn't be a parameter for the regressor but it's hardcoded to 1 and internally used to return reshaped predictions: https://github.com/IQVIA-ML/LightGBM.jl/blob/master/src/predict.jl#L41-L43 so removing it would require to find some decent equivalent in returning these predictions